tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
+#          through libyui-rest-client.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use YuiRestClient::Logger;
+
+sub run {
+    # in Disk Activation page press "ZFCP" button
+    my $disk_activation_selector = $testapi::distri->get_disk_activation_selector();
+    $disk_activation_selector->zfcp();
+
+    # in the ZFCP Device overview press the "Add" button
+    my $zfcp_configuration_overview = $testapi::distri->get_zfcp_configuration_overview();
+    $zfcp_configuration_overview->add();
+
+    # In the Add Device page insert hard wired channel id and press next
+    my $zfcp_add_disk = $testapi::distri->get_zfcp_add_disk();
+    $zfcp_add_disk->set_channel("0.0.fa00");
+    $zfcp_add_disk->next();
+
+    # yui.log from 1st VR showed that it takes a while for
+    # device detection so lets sleep a bit.
+    sleep(8);
+
+    # Now we should be back in ZFCP Device overview page
```

Comment:
here we need a new test module, I would have not bother to check the table because we just rely on the libyui-rest-api doing well its work and then we check the installed system afterward, in other words, this is considered more like what we have called extended scenario.

I would recommend to save this code in some branch and pull out from now, it is overly complicated but I see good intention on that. We already have some function that compares arrays: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a96c5428635e34d07cf2b520e4d557529945f863/lib/utils.pm#L1159
I would definitively leave it for later.